src/target/riscv/riscv-013.c

```diff
@@ -1097,15 +1155,12 @@ static int examine(struct target *target)
 			r->xlen[i] = 64;
 		}
 
-		LOG_DEBUG("hart %d has XLEN=%d", i, r->xlen[i]);
```

Comment:
Can this or something similar be left here? Enumerating the triggers takes a while in simulation, and this one is useful for an early-out if it's really broken.